src/main/resources/application.properties

```diff
@@ -9,3 +9,6 @@ spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
 spring.jpa.open-in-view=false
+
+jwt.expiration=300000
+jwt.secret=hellomates1234567890hellomates1234567890hellomates1234567890
```

Comment:
let's not store secret in main `app.properties`, but pass it in ENV variables just like we do with db username and password